src/shared/logger.ts

```diff
@@ -202,33 +206,35 @@ function makeDateString(type: 'filename' | 'logfile'): string {
     const isFilename: boolean = type === 'filename'
 
     return `${d.getFullYear()}${isFilename ? '' : '-'}` +
-    // String.prototype.padStart() is not available in Typescript...
-    `${padNumber(d.getMonth() + 1)}${isFilename ? '' : '-'}` +
-    `${padNumber(d.getDate())}${isFilename ? 'T' : ' '}` +
-    `${padNumber(d.getHours())}${isFilename ? '' : ':'}` +
-    `${padNumber(d.getMinutes())}${isFilename ? '' : ':'}` +
-    `${padNumber(d.getSeconds())}`
+        // String.prototype.padStart() is not available in Typescript...
+        `${padNumber(d.getMonth() + 1)}${isFilename ? '' : '-'}` +
+        `${padNumber(d.getDate())}${isFilename ? 'T' : ' '}` +
+        `${padNumber(d.getHours())}${isFilename ? '' : ':'}` +
+        `${padNumber(d.getMinutes())}${isFilename ? '' : ':'}` +
+        `${padNumber(d.getSeconds())}`
 }
 
 function padNumber(num: number): string {
     return num < 10 ? '0' + num.toString() : num.toString()
 }
 
-function generateWriteParams(logger: winston.Logger,
-                             level: string,
-                             message: ErrorOrString[],
-                             outputChannel?: vscode.OutputChannel ): WriteToLogParams {
+function generateWriteParams(
+    logger: winston.Logger,
+    level: LogLevel,
+    message: ErrorOrString[],
+    outputChannel?: vscode.OutputChannel
+ ): WriteToLogParams {
     return { logger: logger, level: level, message: message, outputChannel: outputChannel }
 }
 
 interface WriteToLogParams {
     logger: winston.Logger
-    level: string
+    level: LogLevel
     message: ErrorOrString[]
     outputChannel?: vscode.OutputChannel
 }
 
-type ErrorOrString = Error | string
+export type ErrorOrString = Error | string
```

Comment:
Why export?

Reply:
I think for `vsCodeUtils.test.ts`

Reply:
Ctrl + F didn't pick that up, feel free to resolve